packages/devtools_app/test/test_infra/scenes/standalone_ui/editor_service/simulated_editor.dart

```diff
@@ -2,6 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+/// @docImport 'package:devtools_app/devtools_app.dart';
```

Comment:
what is this for? I haven't seen the @docImport annotation before

Reply:
Oh, it's for referencing things in Dartdocs without them being actually imported for code:

https://github.com/dart-lang/sdk/issues/56186

There were some references in the Dartdocs here that I noticed weren't resolved so this fixes them (so you can ctrl+click them).